### ISSUE TITLE
Feature request: nonverbose versions of `csc/cse/cs$/csd`

### ISSUE THREAD

**Issue by soerenwolfers** [lervag/vimtex#1287]
Currently, `cse` changes the height of the command line (if it is at its default value `set cmdheight=1`) to make space for the prompt 

```
Change surrounding environment: <detectedenvironment>
>
```

I find this mildly annoying. Since I imagine most users of vimtex know what the surrounding environment is and what `cse` does to it, I think there should be at least an option to turn off the prompt. 

This would also improve the UI compatibility with vim-surround, which doesn't prompt verbosely when pressing `cs"`, for example.

**Comment by lervag** [comment]
Yes, I can agree that it would be nice to allow a less verbose version. I'll add the option `g:vimtex_echo_verbose_input` which may be disabled to achieve the desired effect.
